build-scripts/build-custom-app.sh

```diff
@@ -0,0 +1,234 @@
+#!/bin/bash
+
+# Prompt the user for the name of the new directory
+read -r -p "Enter the name of the new custom app (use camel-case if required): " custom_app_name
+
```

Comment:
We also want a way to get the acronym for the custom app theme. This must be lowercase and will be used as the folder name for the new theme/tests/in CI etc.
```suggestion
read -p "Input the acronym for the custom app theme (e.g. CPR, CCLW, OEP, MCF...): " theme
theme="$(tr "[:upper:]" "[:lower:]" <<<"${theme}")"
```